### PR TITLE
Add ray build file and update readme

### PR DESCRIPTION
**Pull request by nkwangleiGIT**
Add ray build file and update readme

### CODE REVIEW

File: deploy/charts/arcadia/values.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ global:
 # @param imagePullPolcy ImagePullPolicy
 # @param resources Resources to be used
 controller:
-  image: kubeagi/arcadia:v0.1.0-20231219-874e2b1
+  image: kubeagi/arcadia:v0.1.0-20231230-c8518fb
```
Comment (bjwswang) [inline deploy/charts/arcadia/values.yaml:10]: should bump chart version

File: deploy/llms/Dockerfile.fastchat-worker-ray [hunk 2]
```diff
@@ -0,0 +1,7 @@
+ARG BASE_IMAGE_VERSION="vllm-v0.1.0"
+FROM kubeagi/arcadia-fastchat-worker:${BASE_IMAGE_VERSION}
+
+# Overwrite the python version to pass the version check
```
Comment (bjwswang) [inline deploy/llms/Dockerfile.fastchat-worker-ray:4]: How this overwirte works❓

File: deploy/llms/Dockerfile.fastchat-worker-ray [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+# for now, ray: 2.9.0, python: 3.9.x
+# this utils.py file is from ray 2.9.0 ray-ml image
+COPY utils.py /usr/local/lib/python3.9/dist-packages/ray/_private/utils.py
```
Comment (bjwswang) [inline deploy/llms/Dockerfile.fastchat-worker-ray:7]: So this `utils.py` is doing the version check,right?

File: deploy/charts/arcadia/values.yaml (hunk 1 above)
Comment (nkwangleiGIT) [reply deploy/charts/arcadia/values.yaml:10]: update version to 0.2.1 and appVersionto 0.1.0

File: deploy/llms/Dockerfile.fastchat-worker-ray (hunk 2 above)
Comment (nkwangleiGIT) [reply deploy/llms/Dockerfile.fastchat-worker-ray:4]: the communication between ray head and worker will check the ray and python version, and it must match with x.y.z format, add more comments:
```
# Allow to use environment variable to set ray & python version to pass the version check
# for now, ray: 2.9.0, python: 3.9.x
# this utils.py file is from ray 2.9.0 ray-ml image
# search 'KubeAGI' in utils.py for what's changed
```

File: deploy/llms/Dockerfile.fastchat-worker-ray (hunk 3 above)
Comment (nkwangleiGIT) [reply deploy/llms/Dockerfile.fastchat-worker-ray:7]: yes, in the check_version_info function

**Pull request merged**